config/_default/menus/menus.es-us.yaml

```
@@ -0,0 +1,93 @@
+# Menus - Doks: https://getdoks.org/docs/recipes/menus/
+# Menus | Hugo: https://gohugo.io/content-management/menus/
+
+
+# header-bar/left side: main menus
+main:
+  - name: Descargas
+    url: /downloads/
+    weight: 10
+  - name: Docs
+    url: /docs/
+    weight: 20
+  - name: Rastreo de errores
+    url: https://github.com/Aegisub/Aegisub/issues
+    weight: 30
+  - name: Blog
+    url: /blog/
```

If you don't plan to translate the 3.4.0 announcement post we should redirect to the English blog subpage
```suggestion
    url: ../blog/
```